### ISSUE TITLE
I have a character sprite with 24 rows and 8 columns. How should I set up the generated animation nodes?

### ISSUE THREAD

**Issue by ice-ko** [alextrevisan/LPCAnimatedSprite2D#40]
I have a character sprite with 24 rows and 8 columns. How should I set up the generated animation nodes?

![微信截图_20241013231213](https://github.com/user-attachments/assets/a2616c38-e197-4e7a-a540-837a3ffe8ac3)

As shown in the figure above, how do I set it up when using LPCAnimatedSprite2D?

**Comment by alextrevisan** [comment]
Hi! At the current state it cannot be easily added without editing the plugin source, it's meant to be used with the LPC characters. But I can walk you through, I have the same spriteset from itch.io. I just need some time to process and I wil come back

**Comment by ice-ko** [comment]
> Hi! At the current state it cannot be easily added without editing the plugin source, it's meant to be used with the LPC characters. But I can walk you through, I have the same spriteset from itch.io. I just need some time to process and I wil come back

Is there a demo?

**Comment by alextrevisan** [comment]
Not yet, I need to create all the necessary stuff

**Comment by iMuhammadMustafa** [comment]
Hey Alex, 
Thank you so much for this library

I would like to know if there is a specifications for how the spritesheet should be, I'm trying to use the Longsword from here 

https://sanderfrenken.github.io/Universal-LPC-Spritesheet-Character-Generator/#?body=Body_color_light&head=Human_male_light&weapon=Longsword_longsword

But looking through past issues in your repo, I've found that I need to split that one between the normal sized animation and the oversize animation and add them as different sprites. I was going to do that with Photoshop but I wanted to know if there is something I should keep in mind in terms of image size while doing that.

**Comment by alextrevisan** [comment]
@iMuhammadMustafa if you open the example project there is a precompile longsword example, and a sprite parts example. See what fits you better.
@ice-ko still trying to figure out what can be done

**Comment by ice-ko** [comment]
> @iMuhammadMustafa if you open the example project there is a precompile longsword example, and a sprite parts example. See what fits you better. @ice-ko still trying to figure out what can be done
Thanks, I checked the examples and found that the sprites should be separated and not all put on one picture.

**Comment by iMuhammadMustafa** [comment]
> > Thanks, I checked the examples and found that the sprites should be separated and not all put on one picture.

Yeah me too, I will be editing some sprites as I go forward with my project, I can share them in this repository for anyone who might be interested, or send it to you later. If you also separated them, I would appreciate if you shared them too.

**Comment by alextrevisan** [comment]
You can put all in one picture, in the example they are called precompiled
